### ISSUE TITLE
System.IO.FileNotFoundException on a new dev env

### ISSUE THREAD

**Issue by msasso69** [picoe/Eto#1394]
I had to reinstall my Windows development machine and now I'm not able to run my application nor building Eto code anymore.
What I get when I run my application is:
```
A first chance exception of type 'System.IO.FileNotFoundException' occurred in mscorlib.dll

Additional information: Could not load file or assembly 'System.ComponentModel.TypeConverter, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
```
What I get when I build Eto code is many errors like:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Project Eto.Test is not compatible with net461 (.NETFramework,Version=v4.6.1) / win. Project Eto.Test supports: netstandard1.6 (.NETStandard,Version=v1.6)			0	
```

What I get when I run Eto.Test.Wpf is:
```
System.IO.FileNotFoundException: 'Could not load file or assembly 'System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.'
```
Googling around I think that what is going wrong is about some binding redirects, but I'm kind of stuck.

Please help :)

## Specifications

  - Version: pulled from Eto source code 11th May 2019
  - Platform(s): WPF
  - Operating System(s): Windows 7

**Comment by msasso69** [comment]
solved upgrading VS2017 to 15.9.11

**Comment by SwiTool** [comment]
Sorry but i'm using VScode and getting 
```
System.IO.FileNotFoundException: 'Could not load file or assembly 'System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.'
```
as well. It is already updated, do you know what I can do ?

Btw, if I add in .csproj : 
```
    <PackageReference Include="System.Runtime" Version="4.1.0.0" />
```
`nuget restore` gives : 
```
    NU1605: Detected package downgrade: System.Runtime from 4.3.0 to 4.1.0. Reference the package directly from the project to select a different version.
     testEto.Desktop -> testEto -> NETStandard.Library 1.6.1 -> System.Runtime (>= 4.3.0)
     testEto.Desktop -> testEto -> System.Runtime (>= 4.1.0)
``` 

**EDIT:**

changed `netcoreapp1.6` to `net461` in target and it works.
